src/Video/RenderProgram/SkinRenderProgram.cpp

```diff
@@ -196,8 +196,9 @@ void SkinRenderProgram::Render(const Geometry::Geometry3D* geometry, const Textu
         // Render model.
         glUniformMatrix4fv(modelLocation, 1, GL_FALSE, &modelMatrix[0][0]);
         glm::mat4 normalMatrix = glm::transpose(glm::inverse(viewMatrix * modelMatrix));
```

Comment:
This one is no longer used in the shader.

Reply:
@Ittis  is looking into this.